### DISCUSSION TITLE
Get erro tuning a exe with spacy

### DISCUSSION THREAD

**Discussion by eumateuscandido** [explosion/spaCy#12927] [Help: Coding & Implementations]
I created a virtual environment in pycharm and created an executable from the code below. The executable runs smoothly on my computer, but when I send it to another computer it doesn't work, it shows the error: system could not find the specific file.

I created a exe with: pyinstaller --onefile -w --hiddenimport srsly.msgpack.util --console setup.py

Code: 
def trainmodel():
    def train_model_thread():
        messagebox.showinfo("Treinando Modelo", "Treinando Modelo...")

        with open('./Model/TrainData/TrainData.txt', 'r') as file:
            file_content = file.read()

        data_train = ast.literal_eval(file_content)

        nlp = spacy.blank('pt')
        db = DocBin()
        for text, annot in tqdm(data_train):
            doc = nlp.make_doc(text)
            ents = []
            for start, end, label in annot['entities']:
                span = doc.char_span(start, end, label=label, alignment_mode='contract')
                if span is None:
                    print('Skipping entity')
                    print(annot)
                else:
                    ents.append(span)
            try:
                doc.ents = ents
                db.add(doc)
            except:
                print(text, annot)

        diretorio_executavel = os.getcwd()
        Anotation_folder = os.path.join(diretorio_executavel, 'Model/Anotations')
        if not os.path.exists(Anotation_folder):
            os.makedirs(Anotation_folder)

        db.to_disk('./Model/Anotations/train.spacy')

        os.environ["KMP_DUPLICATE_LIB_OK"] = "TRUE"

        subprocess.run(["python", "-m", "spacy", "init", "fill-config", "Model/base_config.cfg", "config.cfg"])

**Comment by svlandeg** [comment]
Hi,

(note that I converted this to a discussion because the Issue Tracker should only be used for bug reports)

As your code runs fine on your computer, and you're getting an error related to a file on the file system of the other computer, I'm afraid I don't really see how this is related to the spaCy library specifically?
